src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateToArray = date.split(fromFormat[3]);
+  const newDate = [];
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    for (let y = 0; y < fromFormat.length - 1; y++) {
+      if (toFormat[i] === fromFormat[y]) {
```

Comment:
You write `toFormat[i]`, `fromFormat[y]` very often. Let's create variables with meaningful names for them